docs/migration-apiserver-to-crds.md

```diff
@@ -151,3 +159,5 @@ Then you can execute the rollback using this command:
 ```
 helm rollback catalog 1 --cleanup-on-fail --no-hooks
 ```
+
+The migration job PVC with the data about your Service Catalog resources still exists after the rollback. You can perform the migration again to restore your resources.
```

Comment:
Please provide information how to apply those resources, e.g. build a binary or run some docker image? copy data from PVC to local host and execute binary to restore them?

Reply:
done